cmd/porter/server.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"time"
+
+	grpc "get.porter.sh/porter/pkg/grpc"
+	"get.porter.sh/porter/pkg/porter"
+	"get.porter.sh/porter/pkg/signals"
+	"github.com/spf13/cobra"
+)
+
+func buildGRPCServerCommands(p *porter.Porter) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:    "api-server",
+		Short:  "API server",
+		Long:   "Launch API server for porter",
+		Hidden: true, // This is a hidden command and is currently only meant to be used by the porter operator
+	}
+	cmd.Annotations = map[string]string{
+		"group": "api-server",
+	}
+	cmd.AddCommand(buildServerRunCommand(p))
+	return cmd
+}
+
+func buildServerRunCommand(p *porter.Porter) *cobra.Command {
+	opts := porter.ServiceOptions{}
+	cmd := &cobra.Command{
+		Use:   "run",
+		Short: "Run the gRPC server",
+		Long: `Run the gRPC server for porter.
+
+This command starts the gRPC server for porter which is able to expose limited porter functionality via RPC.
+Currently only data operations are supported, creation of resources such as installations, credential sets, or parameter sets is not supported.
+
+A list of the supported RPCs can be found at <link?>
```

Comment:
porter -> Porter (it is the nittiest of nits, but I only included it because I'm going to point out....) -> and <link?> ?